recipe_gdal.Dockerfile

```diff
@@ -0,0 +1,238 @@
+# CentOS 6 with GDAL 3+
+# - includes PROJ v6, ECW J2K 5.5, OPENJPEG 2.3
+# - compatible with pypi GDAL bindings
+# - this docker is not currently compatible with GDAL 2.
+#
+# This dockerfile follows procedures from the offical GDAL dockers
+#   https://github.com/OSGeo/gdal/tree/master/gdal/docker
+#
+# This dockerfile is derived from the manylinux2010 base image, derived from
+# CentOS 6 and already containing many updated build essentials.
+#   https://github.com/pypa/manylinux
+#
+# For the future, the manylinux2010 image may enables a possible portable GDAL
+# that includes internal copies of necessary dependencies and python bindings
+# for a selected python version.
```

Comment:
What version of python are you building for here? I don't see it specified

(also, no 's' on enables)

Reply:
There are no python bindings built here (yet), that's a future idea.  Users will still get their python bindings from pypi.  I'm updating the comments to make this more clear with a working example.